.config/nextest.toml

```diff
@@ -0,0 +1,8 @@
+[profile.ci]
```

Comment:
i wouldnt mind just making it the default like this, we would also not need to mention it anymore.
Or do you think its better to specify for readability?

Reply:
🤷‍♀️ I can see the reasons for both cases.

Reply:
sure, lets leave it